locale/fi/privacy-statement.json

```diff
@@ -0,0 +1,16 @@
+{
+  "title": "Evästekäytännöt",
+  "intro": "",
+  "sections": [
+    {
+      "paragraphs": [
+        {
+          "text": "Designsystem.suomi.fi -sivusto tallentaa käyttäjän vierailusta käyttäjän selaimen muistiin tietoja, joita se vaatii toimiakseen. Näitä tietoja ovat esimerkiksi käyttäjän sijainti sivustolla sekä kielivalinta."
```

Comment:
I this intentionally using the domain name `Designsystem.suomi.fi` 
or should it be instead the page "name", `Suomi.fi Design System`.

_If it uses the domain, maybe lowercasing the first D-letter then._

Reply:
I think it is intentional, since it explicitly refers to the DS-site. I personally think it's clearer this way as well. Also, according to kielitoimisto, the first letter is a capital one even if the sentence starts with a URL (had to check this one):  http://www.kielitoimistonohjepankki.fi/ohje/23

Reply:
Good to know!
Looks weird at first but then it is reasonable when thinking more about it.

Reply:
I Think its ok to have capitalised here since it's referring to the site via address to be specific with which domain these apply to, but is not an actual link? Not sure about this though. I would not change to page name, since this is what was agreed with other stakeholders.